.travis.yml

```diff
@@ -1,24 +1,24 @@
 ---
-language: c
+language: python
+python:
+  - "2.7"
 
 sudo: required
 
 services:
   - docker
 
 cache:
+  pip: true
   directories:
     - $HOME/docker
 
 before_install:
   - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
-  - sudo rm /usr/local/bin/docker-compose
-  - curl -L https://github.com/docker/compose/releases/download/1.16.1/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/1.20.1/docker-compose-`uname -s`-`uname -m` > docker-compose
   - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
-  - sudo apt-add-repository -y ppa:ansible/ansible
-  - sudo apt update
-  - sudo apt install -y ansible
+  - sudo mv ./docker-compose /usr/local/bin/
+  - pip install ansible
```

Comment:
should we specify the ansible version to use?
maybe we can add several versions

Reply:
@juanpaucar  Yes, we can have that. I opened issue #107, PRs are welcomed :)